rllib/utils/policy.py

```diff
@@ -161,6 +164,9 @@ def local_policy_inference(
     env_id: str,
     agent_id: str,
     obs: TensorStructType,
+    reward: float = None,
+    done: bool = None,
+    info: Mapping = None,
```

Comment:
intuitively, why would we need reward and done for inference?

Reply:
I thought the same thing. This becomes an issue for example if we have a prev_reward view requirement.
Initially, I changed the agent collector to fill these will zeros in the agent collector.
But @kouroshHakha said they should be here because if they are view requirements, they ultimately have to be present and I think that's fair.

Reply:
I see. prev_rewards as input to the policy ...
can you please help write a lot of comments about these arguments here? emphasize that these arguments are not necessary unless specific policies really require them as input?
thanks.

Reply:
Done. 
-> It's dictated by the view requirements, even during inference in some cases. To support arbitrary use cases here, like with "`attention_use_n_prev_rewards`", we have to do this I believe.

Reply:
👍